commands.go

```diff
@@ -383,6 +383,9 @@ type Cmdable interface {
 	GeoRadiusStore(ctx context.Context, key string, longitude, latitude float64, query *GeoRadiusQuery) *IntCmd
 	GeoRadiusByMember(ctx context.Context, key, member string, query *GeoRadiusQuery) *GeoLocationCmd
 	GeoRadiusByMemberStore(ctx context.Context, key, member string, query *GeoRadiusQuery) *IntCmd
+	GeoSearch(ctx context.Context, key string, q GeoSearchQuery) *StringSliceCmd
+	GeoSearchWithOptions(ctx context.Context, key string, q GeoSearchWithOptionsQuery) *GeoSearchLocationCmd
```

Comment:
We could rename `GeoSearchWithOptions` to `GeoSearchLocation`. The only problem is that Redis can add a command with such name, but it does not look very likely (they will add another 10 options instead :)